cms/djangoapps/import_from_modulestore/tests/test_tasks.py

```diff
@@ -1,167 +1,73 @@
-    import_course_staged_content_to_library_task,
-    save_legacy_content_to_staged_content_task,
-)
-from openedx.core.djangoapps.content_libraries import api as content_libraries_api
-from openedx.core.djangoapps.content_libraries.api import ContentLibrary
-from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
-from xmodule.modulestore.tests.factories import CourseFactory, BlockFactory
 
-from .factories import ImportFactory
-
-
-class ImportCourseToLibraryMixin(ModuleStoreTestCase):
+class TestImportCourseTask(TestCase):
     """
-    Mixin for setting up data for tests.
+    Tests for the ImportCourseTask class.
     """
 
-    def setUp(self):
-        super().setUp()
-
-        self.library = content_libraries_api.create_library(
-            org=Organization.objects.create(name='Organization 1', short_name='org1'),
-            slug='lib_1',
-            title='Library Org 1',
-            description='This is a library from Org 1',
-        )
-        self.content_library = ContentLibrary.objects.get_by_key(self.library.key)
-
-        self.course = CourseFactory.create()
-        self.chapter = BlockFactory.create(category='chapter', parent=self.course, display_name='Chapter 1')
-        self.sequential = BlockFactory.create(category='sequential', parent=self.chapter, display_name='Sequential 1')
-        self.vertical = BlockFactory.create(category='vertical', parent=self.sequential, display_name='Vertical 1')
-        self.video = BlockFactory.create(category='video', parent=self.vertical, display_name='Video 1')
-        self.problem = BlockFactory.create(category='problem', parent=self.vertical, display_name='Problem 1')
-
-        # self.course2 = CourseFactory.create()
-        # self.chapter2 = BlockFactory.create(category='chapter', parent=self.course, display_name='Chapter 2')
-        self.chapter2 = BlockFactory.create(category='chapter', parent=self.course, display_name='Chapter 2')
-        self.sequential2 = BlockFactory.create(category='sequential', parent=self.chapter2, display_name='Sequential 2')
-        self.vertical2 = BlockFactory.create(category='vertical', parent=self.sequential2, display_name='Vertical 2')
-        self.video2 = BlockFactory.create(category='video', parent=self.vertical2, display_name='Video 2')
-        self.problem2 = BlockFactory.create(category='problem', parent=self.vertical2, display_name='Problem 2')
-
-        self.import_event = ImportFactory(source_key=self.course.id)
-        self.user = self.import_event.user
-
-
-class TestSaveCourseSectionsToStagedContentTask(ImportCourseToLibraryMixin):
-    """
-    Test cases for save_course_sections_to_staged_content_task.
-    """
-
-    def test_save_legacy_content_to_staged_content_task(self):
+    def test_calculate_total_steps(self):
         """
-        End-to-end test for save_legacy_content_to_staged_content_task.
+        Test that calculate_total_steps returns 2 (for staging and importing).
```

Comment:
What is the point of this test if calculate_total_steps has a hard-coded return value of "2"?